code/modules/response_team/ert_outfits.dm

```diff
@@ -85,8 +85,8 @@
 	)
 
 	implants = list(
```

Comment:
Maybe rename this var thats on outfits to `bio_chips`

Reply:
Will do!